spread.yaml

```diff
@@ -90,6 +92,10 @@ path: /home/gopath/src/github.com/snapcore/snapd
 
 exclude:
     - .git
+    - cmd/snap/snap
```

Comment:
It would be nice to be able to tell spread to ignore any files ignored by git already (via .gitignore)